<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>searchPage</class>
 <widget class="QWidget" name="searchPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>598</width>
    <height>399</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>航班购票系统</string>
  </property>
  <property name="windowIcon">
   <iconset resource="image.qrc">
    <normaloff>:/imagSoure/sysu.ico</normaloff>:/imagSoure/sysu.ico</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">#searchPage{
	
	background-image: url(:/imagSoure/flight2.jpg);
	
}</string>
  </property>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>601</width>
     <height>431</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#widget{
	background-image: url(:/imagSoure/flight2.jpg);
}</string>
   </property>
   <widget class="QLabel" name="startSite">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>90</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>出发地：</string>
    </property>
   </widget>
   <widget class="QComboBox" name="StartcomboBox">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>90</y>
      <width>69</width>
      <height>22</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#StartcomboBox{
	border: 1px solid #8f8f91;  /* 设置边框的颜色和宽度 */
    border-radius: 5px;         /* 设置圆角效果 */
    padding: 3px;               /* 设置内边距 */
    background: #f0f0f0;        /* 背景颜色 */
    color: #333;                /* 文字颜色 */
}</string>
    </property>
   </widget>
   <widget class="QLabel" name="endSite">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>90</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>目的地：</string>
    </property>
   </widget>
   <widget class="QComboBox" name="EndcomboBox">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>90</y>
      <width>69</width>
      <height>22</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#EndcomboBox{
    border: 1px solid #8f8f91;  /* 设置边框的颜色和宽度 */
    border-radius: 5px;         /* 设置圆角效果 */
    padding: 3px;               /* 设置内边距 */
    background: #f0f0f0;        /* 背景颜色 */
    color: #333;                /* 文字颜色 */
}</string>
    </property>
   </widget>
   <widget class="mySwapButton" name="myswapButton">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>80</y>
      <width>71</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#myswapButton{
	/*image: url(:/imagSoure/swap1.png);*/
	 background: transparent;
    border: none;  /* 去掉按钮的边框 */
 	
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QDateEdit" name="dateEdit">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>150</y>
      <width>110</width>
      <height>22</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#dateEdit {
    border: 1px solid #8f8f91;  /* 设置边框的颜色和宽度 */
    border-radius: 5px;         /* 设置圆角效果 */
    padding: 3px;               /* 设置内边距 */
    background: #f0f0f0;        /* 背景颜色 */
    color: #333;                /* 文字颜色 */
}

</string>
    </property>
    <property name="dateTime">
     <datetime>
      <hour>0</hour>
      <minute>0</minute>
      <second>0</second>
      <year>2024</year>
      <month>12</month>
      <day>18</day>
     </datetime>
    </property>
    <property name="displayFormat">
     <string>MM-dd</string>
    </property>
    <property name="calendarPopup">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>140</y>
      <width>71</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>出发日期：</string>
    </property>
   </widget>
   <widget class="QPushButton" name="searchPushButton">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>210</y>
      <width>191</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#searchPushButton{
	color: rgb(255, 255, 255);
	background-color: rgb(255, 197, 82);
	border-radius: 10px;
	font-size: 16px;  
}
#searchPushButton:hover{
	color: rgb(255, 255, 255);
	background-color: rgb(255, 163, 87);
	border-radius: 10px;
	
}
#searchPushButton:pressed {
    background-color: rgb(255, 163, 87);
    border: 1px solid #555;  /* 按下时改变边框颜色 */
	border-radius: 10px;
	font-size: 12px;  /* 鼠标按下时，字体缩小 */
}</string>
    </property>
    <property name="text">
     <string>查询</string>
    </property>
   </widget>
   <widget class="QPushButton" name="back1">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>310</y>
      <width>191</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#back1{
	color: rgb(255, 255, 255);
	background-color: rgb(255, 197, 82);
	border-radius: 10px;
	font-size: 16px;  
}
#back1:hover{
	color: rgb(255, 255, 255);
	background-color: rgb(255, 163, 87);
	border-radius: 10px;
	
}
#back1:pressed {
    background-color: rgb(255, 163, 87);
    border: 1px solid #555;  /* 按下时改变边框颜色 */
	border-radius: 10px;
	font-size: 12px;  /* 鼠标按下时，字体缩小 */
}</string>
    </property>
    <property name="text">
     <string>返回</string>
    </property>
   </widget>
   <widget class="QPushButton" name="myTickpushButton">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>260</y>
      <width>191</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#myTickpushButton{
	color: rgb(255, 255, 255);
	background-color: rgb(255, 197, 82);
	border-radius: 10px;
	font-size: 16px;  
}
#myTickpushButton:hover{
	color: rgb(255, 255, 255);
	background-color: rgb(255, 163, 87);
	border-radius: 10px;
	
}
#myTickpushButton:pressed {
    background-color: rgb(255, 163, 87);
    border: 1px solid #555;  /* 按下时改变边框颜色 */
	border-radius: 10px;
	font-size: 12px;  /* 鼠标按下时，字体缩小 */
}</string>
    </property>
    <property name="text">
     <string>我的订单</string>
    </property>
   </widget>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>mySwapButton</class>
   <extends>QPushButton</extends>
   <header location="global">myswapbutton.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="image.qrc"/>
 </resources>
 <connections/>
</ui>
